jiant/shared/model_resolution.py

```diff
@@ -199,4 +208,13 @@ def build_featurization_spec(model_type, max_seq_length):
 
 
 def resolve_tokenizer_class(model_type):
+    """Get tokenizer for a given model architecture.
```

Comment:
Clarify: tokenizer *class*. The tokenizer is still not yet instantiated / loaded.

Reply:
Thanks, done w/ 64b5a99.